packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -150,11 +156,21 @@ At the moment there is no recognized use case for a pixel-width column that woul
 export class MyPixelWidthColumn : TableColumn<...> {
     public constructor() {
         super();
-        this.columnInternals.currentPixelWidth = 100;
+        this.columnInternals.resizingDisabled = true;
```

Comment:
Those are good questions. I didn't really get into those details here since I didn't think they belonged in the Column Width HDL, but I should have included them somewhere. I'll update the Mapping Column HLD with that information.